addons/sale_purchase/models/sale_order.py

```diff
@@ -263,6 +267,7 @@ def _purchase_service_prepare_line_values(self, purchase_order, quantity=False):
             'taxes_id': [(6, 0, taxes.ids)],
             'order_id': purchase_order.id,
             'sale_line_id': self.id,
+            'product_description_variants': line_description,
```

Comment:
Can't use `product_description_variants` in `sale_purchase` as the field is added to the model in the `purchase_stock` module:
https://github.com/odoo/odoo/blob/e41dcde8d66c74c8ad114c76ea6ed9c607e04455/addons/purchase_stock/models/purchase.py#L282

Also I'm not sure it's useful anyway, works perfectly fine without this line, as `name` is the one displayed.